Frameworks/CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -363,10 +359,9 @@ CF_PRIVATE Boolean __CFProcessIsRestricted();
 
 // WINOBJC: The reference platform uses a special compiler option not available to us to 
 // get __CFConstantStringClassReferencePtr filled out with the correct "Class". Since this isn't

```

Comment:
got it

---
In reply to: [78864587](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r78864587) [](ancestors = 78864587,78864285,78862194)